### PR TITLE
Added PHPCS addon

### PR DESCRIPTION
**Pull request by sean-e-dietrich**

### CODE REVIEW

**Comment by sean-e-dietrich** [comment]
Adding as a way to help satisfy https://github.com/docksal/docksal/pull/929

File: phpcs/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+fin phpcs cbf --standard="ExtraStandards,Drupal,DrupalPractice" modules/test_module
+```
+
+### Default Standards Used
+
+For Drupal: Drupal,DrupalPractice
+For WordPres: WordPress
+
+To customize these the `--standard` option could be used.
+Additionally the `PHPCS_STANDARD` variable may set within `.docksal/docksal.env`.
```
Comment (achekulaev) [inline phpcs/README.md]: It says `PHPCS_STANDARD` here and `PHPCS_STANDARDS` in file changes below

File: phpcs/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Default Extensions Used
+
+For Drupal: php,module,inc,install,test,profile,theme,css,info,txt,md
+For WordPress: php
+
+To customize these the `--extension` option could be used.
+Additionally the `PHPCS_EXTENSION` variable may set within `.docksal/docksal.env`.
```
Comment (achekulaev) [inline phpcs/README.md]: `PHPCS_EXTENSIONS`

File: phpcs/phpcs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+fi
+
+# If extension flag is used remove our default.
+EXTENSION_OPT="--extensions=\"${EXTENSIONS}\""
+if [[ "$@" =~ "--extensions=" ]]; then
+  EXTENSION_OPT=""
+fi
+
+# run the command in the container.
+eval fin exec $COMMAND $STANDARD_OPT $EXTENSION_OPT "$@"
```
Comment (achekulaev) [inline phpcs/phpcs]: You should really use quotes around those variables

File: phpcs/phpcs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+fi
+
+# If extension flag is used remove our default.
+EXTENSION_OPT="--extensions=\"${EXTENSIONS}\""
+if [[ "$@" =~ "--extensions=" ]]; then
+  EXTENSION_OPT=""
+fi
+
+# run the command in the container.
+eval fin exec "$COMMAND $STANDARD_OPT $EXTENSION_OPT $@"
```
Comment (achekulaev) [inline phpcs/phpcs]: @sean-e-dietrich it seems there is no need to use `eval` here

Comment (sean-e-dietrich) [reply phpcs/phpcs]: That makes sense actually thanks.

**Review by achekulaev — APPROVED** [review]

**Pull request merged**